taskcat/stacker.py

```diff
@@ -794,237 +670,8 @@ def generate_input_param_values(self, s_parms, region):
-                        gentype = gentype.upper()
-                    else:
-                        gentype = None
-                    if not gentype:
-                        # Set default password type
-                        # A value of PrintMsg.DEBUG will generate a simple alpha
-                        # aplha numeric password
-                        gentype = 'D'
-
-                    if passlen:
-                        if self.verbose:
-                            print("{}AutoGen values for {}".format(PrintMsg.DEBUG, param_value))
-                        param_value = self.genpassword(
-                            passlen, gentype)
-                        _parameters['ParameterValue'] = param_value
-
-                if genaz_re.search(param_value):
-                    numazs = int(
-                        self.regxfind(count_re, param_value))
-                    if numazs:
-                        if self.verbose:
-                            print(PrintMsg.DEBUG + "Selecting availability zones")
-                            print(PrintMsg.DEBUG + "Requested %s az's" % numazs)
-
-                        param_value = self.get_available_azs(
-                            region,
-                            numazs)
-                        _parameters['ParameterValue'] = param_value
-                    else:
-                        print(PrintMsg.INFO + "$[taskcat_genaz_(!)]")
-                        print(PrintMsg.INFO + "Number of az's not specified!")
-                        print(PrintMsg.INFO + " - (Defaulting to 1 az)")
-                        param_value = self.get_available_azs(
-                            region,
-                            1)
-                        _parameters['ParameterValue'] = param_value
-
-                if genaz_single_re.search(param_value):
-                    re_match = genaz_single_re.search(param_value)
-                    az_id = int(re_match.group('az_id'))
-
-                    print(PrintMsg.DEBUG + "Requested 1 az")
-                    print(PrintMsg.DEBUG + "Selecting availability zone %s" % az_id)
-                    param_value = self.get_single_az(region, az_id)
-                    _parameters['ParameterValue'] = param_value
-
-                self.set_parameter(param_key, param_value)
-
-                if getval_re.search(param_value):
-                    requested_key = self.regxfind(getval_re, param_value)
-                    print("{}Getting previously assigned value for {}".format(PrintMsg.DEBUG, requested_key))
-                    param_value = self.get_parameter(requested_key)
-                    print("{}Loading {} as value for {} ".format(PrintMsg.DEBUG, param_value, requested_key))
-                    _parameters['ParameterValue'] = param_value
-
-        return s_parms
+        x = ParamGen(param_list=s_parms, bucket_name=self.get_s3bucket(), boto_client=self._boto_client, region=region, verbose=True)
```

Comment:
the tmp var x is not being used anywhere else. we could:
`return ParamGen(...).param_list`